core/modules/cart/helpers/notifications.ts

```diff
@@ -0,0 +1,50 @@
+import { router } from '@vue-storefront/core/app'
```

Comment:
notifications shouldn't be handled in a core, please keep that in mind. It's purely theme-specific. The function should return result and notification should be sent from theme based on this result

Reply:
I know, even this file will be deleted. For now i'm focusing on the `actions` notifications and everything else will be the next step

Reply:
@filrak good news is that these notifications are just returned and then rendered in the UI - I mean like plain objects: user is to decide whether display them or not